### DISCUSSION TITLE
Is it possible to customize fully using css custom variables without scss for someday?

### DISCUSSION THREAD

**Discussion by capymind** [twbs/bootstrap#38063] [General] [answered]
Hi, I am a totally beginner in client side web development.
Sorry if this question itself seems stupid question.

In docs, to customize some components I have to use SCSS.
But I don't even know what SCSS is and how SCSS works in internals.
I know I have to learn SCSS to fully customize bootstrap options.
(that's my fault...)

But, CSS custom variables also provide a means of customization.
And I love to change color or padding using CSS custom variables.
It's very easy and intuitive and de facto relatively new standard features CSS gives us.

I don't know why some options are customizable only by using SCSS.
Is it possible to use only css custom variables in the future for fully consutmization?

In addition to that, I really appreciate for this wonderful project. 
It's really helpful for designing web front easily and even correctly.

**Comment by mdo** [comment, marked as answer]
We have quite a few CSS variables now in use within Bootstrap, but it's not entirely possible to do what you want yet. Coming in the next major release hopefully.

**Comment by capymind** [comment]
Time flies. CSS and other native functionalities get better and better. Bootstrap is still No1 in prototyping and solid in real services I think. I hope CSS variables get more portions in BS. Happy new year! 🔥
